ts/index.ts

```diff
@@ -1225,9 +1309,26 @@ class Gossipsub extends BasicPubsub {
     // backoff is measured in seconds
     // GossipsubPruneBackoff is measured in milliseconds
     const backoff = constants.GossipsubPruneBackoff / 1000
+    const px: PeerInfo[] = []
+    if (doPX) {
+      // select peers for Peer eXchange
+      const peers = getGossipPeers(this, topic, constants.GossipsubPrunePeers, (xid: string): boolean => {
+        return xid !== id && this.score.score(xid) >= 0
+      })
+      peers.forEach(p => {
+        // see if we have a signed record to send back; if we don't, just send
+        // the peer ID and let the pruned peer find them in the DHT -- we can't trust
```

Comment:
should we add something here like: "not supported at the time of writing in js-libp2p"?